doc/source/overview/high_level.md

```diff
@@ -0,0 +1,508 @@
+# Practical Overview of Explainability
+
```

Comment:
Maybe name the first section "Introduction" or something similar.

Reply:
A table of contents would also be very useful.

Reply:
A good organisation for the first few (separate) sections might be:
- Introduction
- Applications
- Black-box vs. white-box
- Global vs. local explanations

Then we can go into the more detailed "Insights" section where I have some more detailed comments later.